backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/export/SVGExportProcessor.java

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.diagrams.export;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import org.eclipse.sirius.web.diagrams.Diagram;
+import org.eclipse.sirius.web.diagrams.Node;
+import org.eclipse.sirius.web.diagrams.export.api.IExportHandler;
+import org.eclipse.sirius.web.diagrams.export.api.IExportProcessor;
+import org.eclipse.sirius.web.diagrams.export.handlers.EdgeExportHandler;
+import org.eclipse.sirius.web.diagrams.export.handlers.ImageExportHandler;
+import org.eclipse.sirius.web.diagrams.export.handlers.ListExportHandler;
+import org.eclipse.sirius.web.diagrams.export.handlers.ListItemExportHandler;
+import org.eclipse.sirius.web.diagrams.export.handlers.RectangleExportHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Component used to export the elements of the model to their SVG format.
+ *
+ * @author rpage
+ */
+public class SVGExportProcessor implements IExportProcessor {
```

Comment:
Do not use the name `Processor`, it has a specific meaning in our project given our relationship with Reactor